styles/_header.scss

```diff
@@ -0,0 +1,18 @@
+.Tittle {
+    font-family: "Akaya Telivigala";
+    font-weight: 600;
+    grid-area: Tittle;
+    background-color: $SecondaryColor;
+    padding: 1fr 1fr 1fr 1fr;
```

Comment:
fr is only used for CSSgrids columns / rows. replace this with px and it can also be shortened as `padding: 0px`